nimare/meta/cbma/mkda.py

```diff
@@ -1121,7 +1174,7 @@ def _compute_null_approximate(self, ma_maps):
         """
```

Comment:
this doc string should be updated to include sparse arrays

Reply:
Done!